sync/peerset/peer.go

```diff
@@ -49,3 +49,7 @@ func (p *Peer) IsBanned() bool {
 func (p *Peer) IsNodeNetwork() bool {
 	return util.IsFlagSet(p.Flags, PeerFlagNodeNetwork)
 }
+
+func (p *Peer) LastSeenAt() time.Time {
```

Comment:
I don't think we need this function. 

The `LastSeen` property is exported and it is reported in gRPC [here](https://github.com/pactus-project/pactus/blob/f01370fe659c6f9e42cc2f116866562d22561501/www/grpc/network.go#L43)

Reply:
so I think I didn't understand well the purpose of this issue , what feature exactly should implement?

Reply:
It was bad of me. It was fixed here: https://github.com/pactus-project/pactus/commit/f5c78e6fba96c788ba8f7bab76e102ee38408637